.ci-scripts/test-bootstrap.sh

```diff
@@ -12,7 +12,7 @@ rm -rf \
 
 cat ponyup-init.sh | sh -s -- --prefix=/usr/local
 
-export PATH=$HOME/.pony/ponyup/bin:$PATH
+export PATH=$HOME/.local/share/ponyup/bin:$PATH
```

Comment:
the need to hardcode this seems problematic. we should open an issue to look into removing that hardcoding from shell scripts to it doesn't get out of sync with what comes from appdirs. do you agree @Theodus?

Reply:
I agree. I'll add that to the "better unit tests" issue, since using ponytest for this would allow us to use appdirs directly.

Reply:
Oh, this is in `test-bootstrap.sh`. So I guess it may require a different solution.